src/ATDPApplication.ts

```diff
@@ -89,7 +93,8 @@ export abstract class ATDPApplication<T> extends ACLUEWrapper {
     showOptionsLink: false,
     showReportBugLink: true,
     showProvenanceMenu: true,
-    enableProvenanceUrlTracking: true
+    enableProvenanceUrlTracking: true,
+    clientConfig: null
```

Comment:
```suggestion
    clientConfig: {}
```

Why not initializing an empty client config (see also comment below)?